contracts/CertCoin.sol

```diff
@@ -73,13 +80,19 @@ contract CertCoin {
     /// @param _value the max amount they can spend
     /// @param _extraData some extra information to send to the approved contract
     function approveAndCall(address _spender, uint256 _value, bytes _extraData)
-        returns (bool success) {
-        tokenRecipient spender = tokenRecipient(_spender);
+        returns (bool success)
+    {
+        TokenRecipient spender = TokenRecipient(_spender);
         if (approve(_spender, _value)) {
-            spender.receiveApproval(msg.sender, _value, this, _extraData);
+            spender.receiveApproval(
+                msg.sender,
+                _value,
```

Comment:
we need to produce single coding style. e.g '_value' vs 'value'

Reply:
i guess we need to avoid this, because for me as a RoR developer such names look pretty confusing
we usually use variables starting with underscore for memoization